### PR TITLE
Issue #3233: Add new committer procedure.

### PR DESCRIPTION
**Pull request by docwilmot**

### CODE REVIEW

File: procedures/docs/backdrop-committer-procedure.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+* Choose an issue
+  * Make sure it meets [coding standards](https://api.backdropcms.org/coding-standards)
+  * Make sure the logic is sound and simple
+  * Make sure it passes automated tests
+  * Assess wether it needs a new test(s) (usually because it is introducing new functionality)
+  * Test the PR manually in the ZenCI sandbox
+  * Pull down PR verify problem before PR; and test locally too (for exmaple check /admin/reports/dblog for errors or warnings)
+  * Make sure the commit message follows pattern: "Issue #XXXX: Fix type on /admin/blah."
+    * Should be complete sentence
+  * Feature PRs just get merged to `1.x`, bugs and pathces merge to both `1.x` and `cherry-pick` to `1.y.x`
```
Comment (opi) [inline procedures/docs/backdrop-committer-procedure.md]: typo for "patches" ;)

File: procedures/docs/backdrop-committer-procedure.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+
+Backdrop Core Commit Procedure
+==============================
+
+* Choose an issue
+  * Make sure it meets [coding standards](https://api.backdropcms.org/coding-standards)
+  * Make sure the logic is sound and simple
+  * Make sure it passes automated tests
+  * Assess wether it needs a new test(s) (usually because it is introducing new functionality)
```
Comment (laryn) [inline procedures/docs/backdrop-committer-procedure.md]: typo for "whether" ;)

File: procedures/docs/backdrop-committer-procedure.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  * Test the PR manually in the ZenCI sandbox
+  * Pull down PR verify problem before PR; and test locally too (for exmaple check /admin/reports/dblog for errors or warnings)
```
Comment (laryn) [inline procedures/docs/backdrop-committer-procedure.md]: "example"

**Comment by laryn** [comment]
Just submitted a separate PR with these spelling fixes...

**Pull request merged**